bentoml/_internal/utils/analytics/schemas.py

```diff
@@ -199,9 +200,15 @@ class ServeInitEvent(EventMeta):
 class ServeUpdateEvent(EventMeta):
     serve_id: str
     production: bool
-    component: str
+    grpc: bool
```

Comment:
Can we make this field a string to be expandable in the future? `http`, `grpc`, `job`, etc.

Reply:
lets do that